rust/experimental/query_engine/kql-parser/src/kql.pest

```diff
@@ -39,6 +39,11 @@ not_in_token = @{ "!in" }
 not_in_insensitive_token = @{ "!in~" }
 statement_end_token = { &";" }
 
+// Add arithmetic tokens after existing tokens
```

Comment:
nit: Remove this comment it isn't really adding any value 😄

If you are motivated, we could clean up this list of tokens a bit and add some groups like...

```
// Math Tokens
minus_token = { "-" }
plus_token = { "+" }
multiply_token = { "*" }
divide_token = { "/" }
modulo_token = { "%" }
positive_infinity_token = { "+inf" }
negative_infinity_token = { "-inf" }

// Logical Tokens
equals_token = @{ "==" }
equals_insensitive_token = @{ "=~" }
not_equals_token = @{ "!=" }
not_equals_insensitive_token = @{ "!~" }
greater_than_token = @{ ">" ~ !"=" }
greater_than_or_equal_to_token = @{ ">=" }
less_than_token = @{ "<" ~ !"=" }
less_than_or_equal_to_token = @{ "<=" }
and_token = @{ "and" }
or_token = @{ "or" }

// Misc Tokens
...
```

Something like that 😄